src/main/java/step2/domain/LottoNumber.java

```diff
@@ -0,0 +1,44 @@
+package step2.domain;
+
+import java.util.Objects;
+
+public class LottoNumber implements Comparable<LottoNumber>{
+    private static final int MIN_LOTTO_NUMBER = 1;
+    private static final int MAX_LOTTO_NUMBER = 45;
+
+    private final int lottoNumber;
+
+    public LottoNumber(int lottoNumber) {
+        if (lottoNumber < MIN_LOTTO_NUMBER) {
+            throw new IllegalArgumentException("로또 번호는 적어도" + MIN_LOTTO_NUMBER + "이상이어야 합니다.");
+        }
+
+        if (lottoNumber > MAX_LOTTO_NUMBER ) {
+            throw new IllegalArgumentException("로또 번호는 최대" + MAX_LOTTO_NUMBER + "초과해서는 안됩니다.");
+        }
+
+        this.lottoNumber = lottoNumber;
+    }
+
+    public int getLottoNumber() {
+        return lottoNumber;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        LottoNumber that = (LottoNumber) o;
+        return lottoNumber == that.lottoNumber;
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(lottoNumber);
+    }
+
+    @Override
+    public int compareTo(LottoNumber other) {
+        return this.lottoNumber - other.lottoNumber;
```

Comment:
해당 미션에서는 발생하지는 문제이지만 개인적으로 compare시 뺄셈 연산은 권장드리지 않아요 :)
뺄셈 연산 수행시 Integer.MIN_VALUE 혹은Integer.MAX_VALUE의 범위를 넘을경우 overflow가 발생할 수도 있으니
참고하시면 좋을 것 같아요! 

```suggestion
        return Integer.compare(this.lottoNumber, other.lottoNumber);
```